### PR TITLE
update blurb on PHA token bridging mechanism + update links

### PR DESCRIPTION
**Pull request by haochizzle**

### CODE REVIEW

File: packages/config/package.json
Comment (mateuszradomski) [inline packages/config/package.json:1]: Why 😭 

Leave this file alone.

**Review by mateuszradomski — CHANGES_REQUESTED** [review]

File: packages/config/package.json
Comment (haochizzle) [reply packages/config/package.json:1]: lol I'll try again. sorry... to me it doesn't seem like I'm making a lot of changes based on the number of lines changed

Comment (haochizzle) [reply packages/config/package.json:1]: removed the 4 line additions to the packages/config/package.json 

my apologies brother 😭

**Review by luke7211 — CHANGES_REQUESTED** [review]
fix format

**Review by radinat — APPROVED** [review]

**Review by mateuszradomski — APPROVED** [review]

**Pull request merged**